US-EPA-CAMD/easey-ecmps-ui
# feat(5995): only show Configuration Management in select environments

## Related Issues

- [#5995](https://github.com/US-EPA-CAMD/easey-ui/issues/5995#issuecomment-2407872953)

## Main Changes

- Added an environment variable that controls whether the Configuration Management module is available in the application.
- Populated the `manifest-vars[.environment].yml` files with the appropriate values for the variable.

## Steps to Test

- The configuration property `enableConfigurationManagementModule` is enabled by default. Run the application and verify the Configuration Management module is shown in the left-hand navigation, and that it can be accessed directly via its URL route (`http://localhost:<port>/workspace/configuration-management`).
- Set the environment variable `REACT_APP_EASEY_ECMPS_UI_ENABLE_CONFIGURATION_MANAGEMENT_MODULE` to **false**, then restart the application.
- Confirm the Configuration Management module is **NOT** shown in the left-hand navigation, and it can't be otherwise accessed.
